docs/getting_started.rst

```diff
@@ -1,8 +1,71 @@
+===============
 Getting Started
 ===============
 
-This page includes example notebooks intended for new users of `xrtpy`.
+XRTpy is a Python package being developed for the analysis of observations made by the X-Ray Telescope (XRT)
+on the board Hinode spacecraft. This page is intended for new users of `xrtpy`. For more background information about XRT please refer to the `SolarSoft XRT Analysis Guide`_.
+
+
+XRTpy Objects:
+**************
+XRTpy currently offers *Channel*, *Effective Area*, and
+*Temperature Response* classes. These classes allow users
```

Comment:
Change these to reStructuredText links?  Like...`~xrtpy.response.channel.Channel` and so on.